### DISCUSSION TITLE
Looking for Guidance to Contribute to PlutoGrid

### DISCUSSION THREAD

**Discussion by CesareIsHere** [bosskmk/pluto_grid#765] [Q&A]
Hello everyone,

I'm interested in contributing to the development of the PlutoGrid package and I'd like to explore the codebase to better understand how I can help. However, I'm not very familiar with the codebase structure and I would appreciate some guidance to get started.

In particular, I'd like to implement autocomplete as an input type for columns. I already have experience using Flutter and I've seen that a solution is already available, but I believe it could be useful to integrate it directly into PlutoGrid.

I would be grateful if someone could point out the main points of the codebase to examine and suggest where to start.

Thanks in advance for your help and I'm looking forward to contributing to the project.

**Comment by bosskmk** [comment]
I can help with that, but I'm not fluent in English.
When I have time, I'll make a description of the code involved when adding a feature with documentation or video.
Or maybe you can communicate with each other with the help of a translator in a service like discord.
Thank you for your interest in PlutoGrid.
